build/common.sh

```diff
@@ -1007,19 +1007,47 @@ function kube::release::gcs::copy_release_artifacts() {
 }
 
 function kube::release::gcs::publish_latest() {
-  local latest_file_dst="gs://${KUBE_GCS_RELEASE_BUCKET}/${KUBE_GCS_LATEST_FILE}"
+  local -r latest_file_dst="gs://${KUBE_GCS_RELEASE_BUCKET}/${KUBE_GCS_LATEST_FILE}"
 
   mkdir -p "${RELEASE_STAGE}/upload" || return 1
   echo ${KUBE_GCS_LATEST_CONTENTS} > "${RELEASE_STAGE}/upload/latest" || return 1
 
+  local -r release_dir="gs://${KUBE_GCS_RELEASE_BUCKET}/${KUBE_GCS_RELEASE_PREFIX}"
+  if ! gsutil ls ${release_dir}; then
```

Comment:
Nit: Quote the variable: https://google-styleguide.googlecode.com/svn/trunk/shell.xml#Quoting

But also, this should redirect output. See how `kube::release::gcs::ensure_release_bucket` does it, for instance.